package/raphael.js

```diff
@@ -5420,7 +5420,7 @@
         l = 0;
         for (; l < animationElements.length; l++) {
             var e = animationElements[l];
-            if (e.el.removed || e.paused) {
+            if (e.el.removed || e.paused || e.parentEl && e.parentEl.e.paused) {
```

Comment:
Due to shortage of time we are not looking into nested parent animations. We will over-write the entire animation module in the next release.